src/features/pmicControl/npm/npm1300/documentationPmic1300.tsx

```diff
@@ -818,6 +818,25 @@ export const documentation: Documentation = {
                 ],
             },
         ],
+        IBATLIM: [
+            {
+                title: (
+                    <>
+                        <span>IBAT</span>
+                        <span className="subscript">LIM</span>
+                    </>
+                ),
+                content: [
+                    <p key="p1">
+                        For best fuel gauge accuracy it is advised to use
+                        <span>IBAT</span>
+                        <span className="subscript">LIM_LOW</span> for
+                        applications with a lower maximum battery discharge
+                        current than 150mA.
```

Comment:
This text is advising already. No need to state this.
```suggestion
                        For best fuel gauge accuracy, use
                        <span>IBAT</span>
                        <span className="subscript">LIM_LOW</span> for
                        applications with a lower maximum battery discharge
                        current than 150mA.
```

Reply:
@eisl-nordic I am also unsure about the title as @aasinclair  suggested 

Discharge current limit

But all other items are using the label as a the title let me know what is best

Reply:
I assume this will be in the "Charger" card, I think it would be good to add "Battery", so it becomes "Battery Discharge Current Limit", to make it even more clear.